### PR TITLE
changelog 1.0.17

### PR DESCRIPTION
**Pull request by yuhan**
### Summary & Motivation
https://github.com/dagster-io/dagster/blob/yuhan/11-10-changelog_1.0.17/CHANGES.md

### CODE REVIEW

**Comment by yuhan** [comment]
Current dependencies on/for this PR:
* master
  * **PR #10464** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/10464" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/10464?utm_source=stack-comment).

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+# 1.0.17 (core) / 0.16.17 (libraries)
+### New
+
+- With the new `asset_selection` parameter on `@sensor` and `SensorDefinition`, you can now define a sensor that directly targets a selection of assets, instead of targeting a job.
+- `materialize` and `materialize_to_memory` now accept a `raise_on_error` argument, which allows you to determine whether to raise an Error if the run hits an error or just return as failed.
+- (experimental) Dagster now supports multi-dimensional asset partitions, through a new `MultiPartitionsDefinition` object. An optional schema migration enables support for this feature (run via `dagster instance migrate`). Users who are not using this feature do not need to run the migration.
```
Comment (sryza) [inline CHANGES.md:8]:
```suggestion
- (experimental) Dagster now supports multi-dimensional asset partitions, through a new `MultiPartitionsDefinition` object. An optional schema migration enables support for this feature (run via `dagster instance migrate`). Users who are not using this feature do not need to run the migration.
- You can now launch a run that targets a range of asset partitions, by supplying the "dagster/asset_partition_range_start" and "dagster/asset_partition_range_end" tags.
```

Comment (sryza) [reply CHANGES.md:8]: I forgot to add this one ^^, because it made it in late

**Review by sryza — APPROVED** [review]

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+- [dagster-duckdb] In some examples, the duckdb config was incorrectly specified. This has been fixed.
+
+### Breaking Changes
+
+- The behavior of the experimental asset reconciliation sensor, which is accessible via `build_asset_reconciliation_sensor` has changed to be more focused on reconciliation. It now materializes assets that have never been materialized before and avoids materializing assets that are “Upstream changed”. The `build_asset_reconciliation_sensor` API no longer accepts `wait_for_in_progress_runs` and `wait_for_all_upstream` arguments.
+
+### Documentation
+
+- Added documentation outlining [environment variable declaration and usage in Dagster code](https://docs.dagster.io/guides/dagster/using-environment-variables-and-secrets), including how to pass secrets.
+- Fixed a type on Dagster Instance page. Thanks **[@domsj](https://github.com/domsj)**!
```
Comment (erinkcochran87) [inline CHANGES.md:37]: @yuhan Is this supposed to be `type` or should it be `typo`?

**Pull request merged**